src/core/lib/security/credentials/ssl/ssl_credentials.h

```diff
@@ -38,6 +38,11 @@ class grpc_ssl_credentials : public grpc_channel_credentials {
       const char* target, const grpc_channel_args* args,
       grpc_channel_args** new_args) override;
 
+  // TODO(mattstev): Plumb to wrapped languages. Until then, setting the TLS
```

Comment:
Thanks for mentioning this. I was intending to make this change in a follow-up PR, since I'm a little concerned about making this PR too large. Is that ok with you?

Reply:
SG.